### PR TITLE
added test for updating value set versions as these were nto being pr…

### PR DESCRIPTION
**Pull request by edeyoung**
…operly handled previously.

### CODE REVIEW

**Review by lukoktonos — DISMISSED** [review]
Looks good.

File: lib/tasks/cql.rake [hunk 1]
```diff
@@ -84,5 +84,43 @@ namespace :bonnie do
...
+
+            elms.each do |elm|
+
+              if elm['library'] && elm['library']['valueSets'] && elm['library']['valueSets']['def']
+                elm['library']['valueSets']['def'].each do |value_set|
+                  db_value_sets = HealthDataStandards::SVS::ValueSet.where(user_id: user.id, oid: value_set['id'])
+
+                  db_value_sets.each do |db_value_set|
+                    if value_set['version']
+                      if db_value_set.version == "N/A"
```
Comment (jbradl11) [inline lib/tasks/cql.rake]: This isn't critical, but I would suggest making this one if statement. Using '&&'

**Review by jbradl11 — APPROVED** [review]
Looks good.

**Review by lukoktonos — APPROVED** [review]
Looks good.

**Pull request merged**